packages/components/src/Typeahead/Typeahead.snapshot.test.js

```diff
@@ -54,9 +58,7 @@ describe('Typeahead', () => {
 			};
 
 			// when
-			const wrapper = renderer
-				.create(<Typeahead {...props} />)
-				.toJSON();
+			const wrapper = renderer.create(<Typeahead {...props} />).toJSON();
```

Comment:
where are the snap?

Reply:
Why would you want to see it modified by this test ?

Reply:
Dunno, right.